desktop/AutoStartManager.ts

```diff
@@ -62,10 +65,10 @@ class AutoStartManager {
     );
   }
 
-  static isEnabled = () => AutoStartManager.service.isEnabled();
+  static isEnabled = () => StoreService.get(IS_AUTO_START_ENABLED);
```

Comment:
Since we bring back this flag we need to sync the actual status with the flag in the settings inside the `init` method: if flag is `false` then disable autostart on start, if `true` — enable it.